test/browser/pages/DVADetailsEntryPage.js

```diff
@@ -136,6 +136,17 @@ exports.DVADetailsEntryPage = class PlaywrightDevPage {
     this.serviceDescription = this.page.locator(
       'xpath=//*[@id="main-content"]/div/div/div'
     );
+
+    this.consentSectionSentenceOneDVA = this.page.locator(
+      'xpath=//*[@id="consentDVACheckbox-hint"]/p[1]'
+    );
+
+    this.consentSectionSentenceTwoDVA = this.page.locator(
+      'xpath=//*[@id="consentDVACheckbox-hint"]/p[2]'
+    );
+    this.privacyPolicyDVALink = this.page.locator("a", {
+      hasText: "hysbysiad preifatrwydd DVA (agor mewn tab newydd)",
```

Comment:
Can we do this with xpath or an id we want to be able to reuse this outside of the welsh check